packages/core/src/lib/modal.svelte

```diff
@@ -74,7 +74,9 @@ $: if (typeof document !== 'undefined') {
 }
 
 onDestroy(() => {
-  document.body.classList.remove('overflow-hidden');
+  if (typeof document !== 'undefined') {
+    document.body.classList.remove('overflow-hidden');
+  }
```

Comment:
fly by fix -- was breaking the playground

Reply:
callout by @micheal-parks to use the return value of `onMount` instead, because `onDestroy` runs on the server